### DISCUSSION TITLE
Hundereds of MBI5153 panels

### DISCUSSION THREAD

**Discussion by drvkmr** [LAutour/ESP32-HUB75-MatrixPanel-DMA-ICN2053#8] [Q&A]
I've got my hands on hundereds of MBI5153 panels from unknown manufacturer. They were about to get binned but I found them in the right time.
However I am struggling to get them to work - open to ideas on how to go about it. I am happy to send a couple to anyone who wants to try it out as well as it can potentially save hundereds of them.
[https://photos.app.goo.gl/D8DjhibQpYi4sfwB6](url)

**Comment by lautour2** [comment]
This number of signal outputs is still in the process of development (esp32-s2 for sure, esp32 - if there are enough outputs)

**Reply by drvkmr** [reply]
I am using esp32-s3 which has enough outputs. But even if I only use 2 lines of RGB (R1,G1,B1,R2,B2,G2) it should still show something right?

**Reply by lautour2** [reply]
esp32-s3 can output only 16 bits in parallel via DMA. We'll have to think about another way to switch the screen lines.

**Reply by drvkmr** [reply]
Ah ok. Didn't know that. Do you think DMA is not the way to go for this panel then?

**Reply by lautour2** [reply]
I have ideas, I need to check them out.

**Reply by drvkmr** [reply]
Ok, thanks. Let me know if I can help in any way.
